modules/virt-cluster-resource-requirements.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * virt/install/preparing-cluster-for-virt.adoc
+
+[id="virt-cluster-resource-requirements{context}"]
+= Additional hardware requirements for {VirtProductName}
+
+
+{VirtProductName} is an add-on to {product-title} and imposes additional overhead that you need to take into account when planning a cluster. Each cluster machine must meet the following hardware requirements in addition to the {product-title} requirements.
+
+.Memory
+
+----
+Memory overhead per virtual machine = (1.002 * requested memory) + 146 MiB  \
+                + (8 * number of vCPUs) + (16 MiB * number of graphics devices)
+----
+
+Add 1 GiB to the above result if your environment includes a Single Root I/O Virtualization (SR-IOV) network device or a Graphics Processing Unit (GPU).
+
+----
+Cluster memory overhead = 2179 MiB + (366 MiB * number of nodes) \
```

Comment:
This part feels confusing. Perhaps we could split it up differently for clarity/accuracy:

-----------

infrastructure nodes:
158 MiB for each infrastructure node -- plus:
21 CNV infrastructure containers that total 2179 MiB of RAM will be spread across all infrastructure nodes (masters).

--------------
The memory overhead of worker nodes is:
366 MiB per node

-------

The memory overhead of each Virtual Machine:
Memory overhead per virtual machine = (1.002 * requested memory) + 146 MiB  \
                + (8 * number of vCPUs) + (16 MiB * number of graphics devices)